File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    var arr = s.split(" ");
    for(let i=0; i<arr.length;i++){
        for(let j=0; j<arr[i].length; j++){
            if(j%2==0){
                answer += arr[i][j].toUpperCase();
            }else{
                answer += arr[i][j].toLowerCase();
            }
        }
        answer += " ";
    }
    return (answer.slice(0,answer.length-1));
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int idx = 0;
        
        for(int i=0; i<s.length(); i++){
            if(s.charAt(i)==' '){
                answer += " ";
                idx = 0;
                continue;
            }
            
            if(idx%2==0){
                answer += ("" + s.charAt(i)).toUpperCase();
            }else{
                answer += ("" + s.charAt(i)).toLowerCase();
            }
            idx++;
        }
        
        return answer;
    }
}